### ISSUE TITLE
Stage1 training crashes on eval

### ISSUE THREAD

**Issue by gaoxiao** [OpenMotionLab/MotionGPT#22]
Command line used: 
python -m train --cfg configs/config_h3d_stage1.yaml --nodebug

Error:
torch._C._LinAlgError: linalg.svd: (Batch element 0): The algorithm failed to converge because the input matrix contained non-finite values.

<img width="1057" alt="image" src="https://github.com/OpenMotionLab/MotionGPT/assets/1241824/2e52d89f-b2b2-49db-ac91-69cef545ff7d">

**Comment by billl-jiang** [comment]
Hi, @gaoxiao 

This error is most likely caused by the incorrectly processed dataset. We recommend re-processing the HumanML3D dataset following the instructions in their [official repo](https://github.com/EricGuo5513/HumanML3D). Hope this can help you solve the problem.

**Comment by gaoxiao** [comment]
> Hi, @gaoxiao
> 
> This error is most likely caused by the incorrectly processed dataset. We recommend re-processing the HumanML3D dataset following the instructions in their [official repo](https://github.com/EricGuo5513/HumanML3D). Hope this can help you solve the problem.


Replacing the test data with validation data fixed this issue, confirming that the test data is indeed corrupted.

**Comment by kruzel** [comment]
how do you change the test data with validation data?

**Comment by kruzel** [comment]
Hi, @gaoxiao
can you tell me how to resolve the dataset corruption problem?

> Replacing the test data with validation data fixed this issue, confirming that the test data is indeed corrupted.

**Comment by zybermonk** [comment]
Hi, I was facing the same issue.

- I found a couple files in the test set had non-finite values ( Files '007975', 'M007975').
- I removed them and their corresponding motion file counterparts.
- Yet, the same issue continues.

However, when I change the SPLIT in, # Evaluating Configuration
```
EVAL:
  BATCH_SIZE: 32 # Evaluating Batch size
  SPLIT: test -->  val
```
**The training works fine.** Confirming the experience from @gaoxiao above.
I would like to know why the Evaluation Configuration uses 'test' SPLIT by default? is this recomennded?
